python/src/energuide/extracted_datatypes.py

```diff
@@ -109,10 +94,8 @@ class _WaterHeating(typing.NamedTuple):
 
 
 _RSI_MULTIPLIER = 5.678263337
-_CFM_MULTIPLIER = 2.11888
 _FEET_MULTIPLIER = 3.28084
 _FEET_SQUARED_MULTIPLIER = _FEET_MULTIPLIER**2
-_MILLIMETRES_TO_METRES = 1000
```

Comment:
`_MILLIMETRES_TO_METRES` was just something that should have been removed before?

Reply:
Yes